src/LinuxTracing/CMakeLists.txt

```diff
@@ -96,3 +96,12 @@ target_link_libraries(LinuxTracingTests PRIVATE
         GTest::Main)
 
 register_test(LinuxTracingTests)
+
+add_executable(ThreadDepedencyTestApp)
+
+target_sources(ThreadDepedencyTestApp PRIVATE
+        ThreadDepedencyTestApp.cpp)
+
+target_link_libraries(ThreadDepedencyTestApp PRIVATE
+        OrbitBase)
+        
```

Comment:
This change should be part of the other pull request which is also adding the `ThreadDependencyTestApp.cpp` file.